templates/prWaitingRoom.html

```diff
@@ -154,23 +154,23 @@ <h2 class="mdc-dialog__title" id="my-dialog-title">Join room</h2>
     </div>
     <div class="mdc-dialog__scrim"></div>
 </div>
-<script src="app.js"></script>
+<script src="../FirebaseRTC/public/app.js"></script>
     <!-- jQuery -->
-    <script src="/temp/assets/js/jquery-2.1.0.min.js"></script>
+    <script src="../temp/assets/js/jquery-2.1.0.min.js"></script>
 
     <!-- Bootstrap -->
-    <script src="/temp/assets/js/popper.js"></script>
-    <script src="/temp/assets/js/bootstrap.min.js"></script>
+    <script src="../temp/assets/js/popper.js"></script>
+    <script src="../temp/assets/js/bootstrap.min.js"></script>
 
     <!-- Plugins -->
-    <script src="/temp/assets/js/owl-carousel.js"></script>
-    <script src="/temp/assets/js/scrollreveal.min.js"></script>
-    <script src="/temp/assets/js/waypoints.min.js"></script>
-    <script src="/temp/assets/js/jquery.counterup.min.js"></script>
-    <script src="/temp/assets/js/imgfix.min.js"></script> 
+    <script src="../temp/assets/js/owl-carousel.js"></script>
+    <script src="../temp/assets/js/scrollreveal.min.js"></script>
+    <script src="../temp/assets/js/waypoints.min.js"></script>
+    <script src="../temp/assets/js/jquery.counterup.min.js"></script>
+    <script src="../temp/assets/js/imgfix.min.js"></script> 
     
     <!-- Global Init -->
-    <script src="/temp/assets/js/custom.js"></script>
+    <script src="../temp/assets/js/custom.js"></script>
```

Comment:
For all of the script tags above this comment, update the srcs to match the new naming system. The project directory is the root. Refer to the first comment on this file for an example.